apps/src/slider.js

```diff
@@ -212,14 +216,41 @@ Slider.prototype.getValue = function () {
  * @param {number} value New value.
  */
 Slider.prototype.setValue = function (value) {
-  this.value_ = Math.min(Math.max(value, 0), 1);
+  value = Math.min(Math.max(value, 0), 1);
+  this.defaultValue_ = value;
+
+  var override = this.getValueOverride_();
+  if (override && value === override.from) {
+    value = override.to;
+  }
+  this.value_ = value;
+
   var x = this.KNOB_MIN_X_ +
       (this.KNOB_MAX_X_ - this.KNOB_MIN_X_) * this.value_;
   this.knob_.setAttribute('transform',
       'translate(' + x + ',' + this.KNOB_Y_ + ')');
 };
 
 /**
+ * Read the value override from sessionStorage.
+ * @return {Object} containing the value to override and the new value
+ */
+Slider.prototype.getValueOverride_ = function () {
+  var override = sessionStorage.getItem(OVERRIDE_STORAGE_KEY);
+  if (override) {
+    return JSON.parse(override);
+  }
+  return null;
```

Comment:
nit: leaving out this return means this method would return `undefined`, which feels semantically better anyway